recipes/pyopencl/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyopencl" %}
+{% set version = "2016.2" %}
+{% set sha256 = "1b94540cf59ea71a3ef234a8f1d0eb2b4633c112f0f554fb69e52b4a0337d82b" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: True  # [not (linux or osx)]
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - cffi
+        - ocl-icd  # [linux]
+    run:
+        - python
+        - numpy
+        - pytools
+        - six
+        - mako
```

Comment:
It only gets used at runtime. The main deal (to me) about OpenCL is that you can generate code at run time and run it god-knows-where (GPU, CPU, whatever). Mako is one of the ways it uses to do that.